Started by timer
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Running on [8mha:////4C8PgMHPegRpCg8+HpIiIHK02uxvuQktJylkqRHkJagNAAAAox+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzeEgYJ/eT83ILSktQiICOvpCg/Jye1SNfSUh8ABjlyb80AAAA=[0mcontroller-99 in /root/.jenkins/workspace/cyp-test-nic-sriov-5times
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (rm project dir)
[8mha:////4EWjgIQS1R18e39T6LdOu3ZpSMSXMburmKwxJHd8e163AAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gwExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jcChmMxwAAAAA==[0m[Pipeline] script
[8mha:////4NOJ/YVWDlnnBpjDdLKlLZkF5LrR387vblsx7fWhZkPlAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0BAkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfpX/cvwAAAAA==[0m[Pipeline] {
[8mha:////4MAm0moKTc8Eo2PcB0LgkHNHI7Xtf5Poej2Mf/l8xh/lAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaoyVLaHxBMgiwpL/WWDdrTyRV/MOEjexcpJJ5k3zXm9Y5wQnSpZ7g73DrB2PYbR18YlSfws0cU9XrgkzBcOlmSR15rygpGJgCWtgJWBjUAfKDm2BrfDqodqg0LaXkup3FNC4boAnMFHNRaXyhTnB7j52mfDn+SuAORZg+9pD/AAeoCAqvAAAAA==[0m[Pipeline] sh
+ ls /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester
ls: cannot access '/root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester': No such file or directory
[8mha:////4MftwC6NxviNjDdpfna9VBlJkoMp8EhRTGL2zcTKdEZ0AAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKBDglR0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77aCbP8Brmfz270AAAA=[0m[Pipeline] }
[8mha:////4Kr9K7llyIIfXzIi7fTbyam3A+VyjB0I1hAyH+fEbXLCAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiIMlYVjZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0gL0tBtv8Aj1Msdr0AAAA=[0m[Pipeline] // script
[8mha:////4FI2+bSiptyXTIERd0s3FW0AIUAVzDIQC15QCQE4LYPvAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRFDSIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPyRmLRr0AAAA=[0m[Pipeline] }
[8mha:////4EIF7pCgSAhTUJ+YeVF144RCQBElcAy/oq5IQm3P5r/FAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRQlSIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPw2Hi4r0AAAA=[0m[Pipeline] // stage
[8mha:////4O65h0jm98EO51SGQmOdf6FpfngMUfFS1Hkxj+qIATCfAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjpHSIitZywwtMYowT6y7YF5KKF/E1/oAhEhVb7ay0mucLyhThQNHJzmLvMTVeDmF0ucmJYn8JNMmOzrIhTBSs1HbS1NrjgprYwhJRwErB2mITKHl0DBvVmbupgkFXnTjmba+g8O0NHiBUNrOJ/IU5wvY6tonw5/krgHlgEDVDsas/x/INoKa7ysEAAAA=[0m[Pipeline] stage
[8mha:////4Cwh2bfnyXtsNeYvUrnMhkgZn3DOCE3ZIocZCmHhDj1kAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUSDAhpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD5WQ+PfCAAAA[0m[Pipeline] { (git clone)
[8mha:////4CFOXQFtiiA+7tA4CXv18jhcFszpIgsA6m3wBj5vXwoxAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiCmgQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHN7LjSKW+AAAA[0m[Pipeline] sh
+ git clone -b develop https://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@github.com/Jiz1028/applications.validation.network-platform.npxtester.git npxtester
Cloning into 'npxtester'...
[8mha:////4JxnGUheoVbdA4jH7w1zuxcsCY2poKXOhDnLMn7NKJYFAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiCkSBqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQGOY4V7vgAAAA==[0m[Pipeline] }
[8mha:////4D4+5H4RzROPZTW0wBHD+XjMhoO5mPw9EWUJAbCIJpuGAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiJAokRJXWcsMLTGyME+vO2A5JxYv4Gn8gEImKLVY728zzBcsU4cjRYmuoc5Qah8H3dlo4cOwungds+YwNU2JvUJpBsjb1jJKzgTlFCQsBK0ON5+TIZliLVt1V5RXZ6pTj9B0ElE7f4AGFmMxZxfyFMcLm2uvE9PP8FcAYQoZyu//U7g3N4dT9vgAAAA==[0m[Pipeline] // stage
[8mha:////4AJAGqZwaYqyNNZtbc8g6ZE112gHcOMJEUqcQ3uLTkOfAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjJBqEqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A361JXbBAAAA[0m[Pipeline] stage
[8mha:////4PP68w9I/uoPNBdRzp82S+tTLGAqcpDjYGyOG/BwZs8HAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycogU2BDqxBp14QShCSFp9H9JUtKJE3E17kCgEhMeLNuS9Z4vqIKHhrxmVmFvMHSGDW7UObFEvr84SszSmXWEgZxirUotSXWca0tRwayihAWHpcLOUTCoI6y4FXdRO4G6PkWftwOH0sgbPKDgmRyFj98yeVhfRxkIf5y/AJiGCOV2n223+TyrN7xWSV3CAAAA[0m[Pipeline] { (pre-exec-set-args)
[8mha:////4BsNiMtY/8UtmXAIxp6NV4VtqKiaHUkJeJGRvFkfJcU+AAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICtIgqreWGF5jYGCfWnbEdkooX8TX+QCASFVusdraZ5wuWKcKRo8XWUOcoNQ6D7+20cODYXTwP2PIZG6bE3qA0g2Rt6hklZwNzihIWAlaGGs/Jkc2wFq26q8orstUpx+k7CCidvsEDCjGZs4r5C2OEzbXXienn+SuAMWQo99tP7cIbaOP4NL4AAAA=[0m[Pipeline] sh
+ python /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/conf/change_args.py '{"drive_info":{"drive_version":"1.7.11"},"dpdk_dir":"/opt/APP/utility/DPDK_install/dpdk-21.02/","vm_image_conf":{"image1":"uvm1.img","image2":"uvm2.img","image3":"rhel84rt.img","image4":"rhel84rt2.img","image_dir":"/home/image"},"sut_conf":{"host":"10.239.182.209","pf1":"ens785f2","pf1_ipv6":"fe80::42a6:b7ff:fe72:ccca","os":"RHEL"},"sut2_conf":{"host":"10.239.182.79","pf1":"ens787f2","pf1_ipv6":"fe80::42a6:b7ff:fe82:eb6"}}'
[8mha:////4AyFPX+UZcZf/6aTNXdj+lWE0Z2b4chbTZ9e7eCHFoeRAAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKC0iIqWisNLzCJMXasO8d2cCpexNf4A4FIVGyx2tlmni9YxwBHDhqtot5QbA16N+p5YebQXx1ntHzBlimyU9io3HCnTgs2nBQsKUpYCdgoah1HQzrBVlh5l5WTpKtzCvN3EFCaboAHFGI2JxnSF6YAu9vYRaaf568AJu8TlHX9qf0b7E/nQb4AAAA=[0m[Pipeline] }
[8mha:////4AFg/aWh/jHSmrlH6sy1kKjz0TFBfVRN+miPQW2LyX/uAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIioEIoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDQMEcrdPtf28AEud0rYvgAAAA==[0m[Pipeline] // stage
[8mha:////4Bhd+KV1EsVM3bnXBHnMClLv0EnBj/3vPQbr+7HWEbmAAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjoXSIitZywwtMYowT6y7YF5KKF/E1/oAhEhVb7ay0mucLyhThQNHJzmLvMTVeDmF0ucmJYn8JNMmOzrIhTBSs1HbS1NrjgprYwhJRwErB2mITKHl0DBvVmbupgkFXnTjmba+g8O0NHiBUNrOJ/IU5wvY6tonw5/krgHlgEDVDsas/x/INPbxT+8EAAAA=[0m[Pipeline] stage
[8mha:////4P13TIuP26wSoF8WyLGiU1qV0Q6Z+QylTska1uSfBb7mAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUQjAhpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD/hYjrHCAAAA[0m[Pipeline] { (run nic case)
[8mha:////4FM3oozjxgRBqafqHchkdSyZ1a7wU/MTBVKRbtZrSwmtAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiIUGDqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDhnK7+9Q+vAEIWoEpvgAAAA==[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics -s -m nicSriov5times --capture=tee-sys --html=/root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-3.1.1
collected 16 items / 15 deselected / 1 selected

npxtester/Tests/testNics/test_network_sriov_stress.py 2022-03-10 02:52:33 - NicTestFixture.py - [line:28] - root - INFO - ============================ Starting to run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm2.img-uvm1.img] case ============================
2022-03-10 02:52:33 - test_network_sriov_stress.py - [line:31] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-03-10 02:52:33 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-10 02:52:34 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-10 02:52:34 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-03-10 02:52:34 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-10 02:52:34 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-10 02:52:34 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-10 02:52:34 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-03-10 02:52:37 - sshInstance.py - [line:46] - root - INFO - modinfo ice | grep ^version:
2022-03-10 02:52:37 - sshInstance.py - [line:50] - root - INFO - version:        1.7.11

2022-03-10 02:52:45 - sshInstance.py - [line:46] - root - INFO - rmmod ice
2022-03-10 02:52:56 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 02:53:04 - sshInstance.py - [line:46] - root - INFO - modprobe ice
2022-03-10 02:53:09 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 02:53:17 - sshInstance.py - [line:46] - root - INFO - modinfo ice
2022-03-10 02:53:17 - sshInstance.py - [line:50] - root - INFO - filename:       /lib/modules/4.18.0-193.19.1.rt13.70.el8_2.x86_64/updates/drivers/net/ethernet/intel/ice/ice.ko
firmware:       intel/ice/ddp/ice.pkg
version:        1.7.11
license:        GPL v2
description:    Intel(R) Ethernet Connection E800 Series Linux Driver
author:         Intel Corporation, <linux.nics@intel.com>
rhelversion:    8.2
srcversion:     4AE55166A76503F69ADCE95
alias:          pci:v00008086d0000151Dsv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Fsv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Esv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Dsv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Csv*sd*bc*sc*i*
alias:          pci:v00008086d0000189Asv*sd*bc*sc*i*
alias:          pci:v00008086d00001899sv*sd*bc*sc*i*
alias:          pci:v00008086d00001898sv*sd*bc*sc*i*
alias:          pci:v00008086d00001897sv*sd*bc*sc*i*
alias:          pci:v00008086d00001894sv*sd*bc*sc*i*
alias:          pci:v00008086d00001893sv*sd*bc*sc*i*
alias:          pci:v00008086d00001892sv*sd*bc*sc*i*
alias:          pci:v00008086d00001891sv*sd*bc*sc*i*
alias:          pci:v00008086d00001890sv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Esv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Dsv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Csv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Bsv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Asv*sd*bc*sc*i*
alias:          pci:v00008086d0000159Bsv*sd*bc*sc*i*
alias:          pci:v00008086d0000159Asv*sd*bc*sc*i*
alias:          pci:v00008086d00001599sv*sd*bc*sc*i*
alias:          pci:v00008086d00001593sv*sd*bc*sc*i*
alias:          pci:v00008086d00001592sv*sd*bc*sc*i*
alias:          pci:v00008086d00001591sv*sd*bc*sc*i*
depends:        auxiliary
name:           ice
vermagic:       4.18.0-193.19.1.rt13.70.el8_2.x86_64 SMP preempt mod_unload modversions 
parm:           debug:netif level (0=none,...,16=all) (int)
parm:           fwlog_level:FW event level to log. All levels <= to the specified value are enabled. Values: 0=none, 1=error, 2=warning, 3=normal, 4=verbose. Invalid values: >=5
 (ushort)
parm:           fwlog_events:FW events to log (32-bit mask)
 (ulong)

2022-03-10 02:53:22 - test_network_sriov_stress.py - [line:388] - root - INFO - 

********************************************************Stress Disable and Enable for 1 time when VF is Assigned VM guest.*************************************************************


2022-03-10 02:53:25 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -s | grep ens785f2 | awk '{print $1;}'
2022-03-10 02:53:26 - sshInstance.py - [line:50] - root - INFO - 0000:4b:00.2

2022-03-10 02:53:31 - test_network_sriov_stress.py - [line:108] - root - INFO - pf_id was found: 0000:4b:00.2

2022-03-10 02:53:31 - test_network_sriov_stress.py - [line:396] - root - INFO - format_pf_id is 0000\:4b\:00.2

2022-03-10 02:53:31 - test_network_sriov_stress.py - [line:399] - root - INFO - pf_id_key: 4b
2022-03-10 02:53:37 - sshInstance.py - [line:94] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Thu Mar 10 01:37:51 2022 from 10.239.182.99
[root@npx-cyp7-rh82 ~]# ifconfig ens785f2 192.168.10.10
[root@npx-cyp7-rh82 ~]# 
2022-03-10 02:53:40 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens785f2
2022-03-10 02:53:43 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-10 02:53:43 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 02:53:48 - test_network_sriov_stress.py - [line:122] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-10 02:53:51 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-10 02:53:51 - sshInstance.py - [line:50] - root - INFO - 4b:11.0
4b:11.1
4b:11.2
4b:11.3

2022-03-10 02:53:56 - test_network_sriov_stress.py - [line:136] - root - INFO - Succssfully Gotten vf_id list:

2022-03-10 02:53:59 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-10 02:53:59 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 02:54:07 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.0
2022-03-10 02:54:08 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 02:54:13 - test_network_sriov_stress.py - [line:169] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-10 02:54:16 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.1
2022-03-10 02:54:17 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 02:54:22 - test_network_sriov_stress.py - [line:169] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-10 02:54:22 - test_network_sriov_stress.py - [line:420] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-10 02:54:25 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-10 02:54:25 - sshInstance.py - [line:50] - root - INFO - 0000:4b:11.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-03-10 02:54:30 - test_network_sriov_stress.py - [line:431] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-10 02:54:33 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-10 02:54:33 - sshInstance.py - [line:50] - root - INFO - no

2022-03-10 02:54:38 - test_network_sriov_stress.py - [line:196] - root - INFO - Default network is active? no

2022-03-10 02:54:38 - test_network_sriov_stress.py - [line:205] - root - INFO - virsh net-start default
2022-03-10 02:54:41 - sshInstance.py - [line:46] - root - INFO - virsh net-start default
2022-03-10 02:54:42 - sshInstance.py - [line:50] - root - INFO - Network default started


2022-03-10 02:54:47 - test_network_sriov_stress.py - [line:209] - root - INFO - Successfully started network default.
2022-03-10 02:54:51 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
2022-03-10 02:54:52 - sshInstance.py - [line:71] - root - INFO - WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-10 02:54:54 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 02:54:57 - test_network_sriov_stress.py - [line:231] - root - INFO - Successfully created a VM: VMtest1
2022-03-10 02:55:01 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-10 02:55:04 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 02:55:07 - test_network_sriov_stress.py - [line:231] - root - INFO - Successfully created a VM: VMtest2
2022-03-10 02:55:10 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-10 02:55:10 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 02:55:15 - test_network_sriov_stress.py - [line:248] - root - ERROR - not found IP of VMtest1
2022-03-10 02:56:18 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-10 02:56:18 - sshInstance.py - [line:50] - root - INFO - 192.168.122.158/24

2022-03-10 02:56:23 - test_network_sriov_stress.py - [line:243] - root - INFO - Successfully got VMtest1 IP: 192.168.122.158/24

2022-03-10 02:56:26 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-10 02:56:26 - sshInstance.py - [line:50] - root - INFO - 192.168.122.16/24

2022-03-10 02:56:31 - test_network_sriov_stress.py - [line:243] - root - INFO - Successfully got VMtest2 IP: 192.168.122.16/24

2022-03-10 02:56:31 - test_network_sriov_stress.py - [line:448] - root - INFO - ip of VM1: 192.168.122.158
2022-03-10 02:56:31 - test_network_sriov_stress.py - [line:449] - root - INFO - ip of VM1: 192.168.122.16
2022-03-10 02:56:35 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.158
PING 192.168.122.158 (192.168.122.158) 56(84) bytes of data.
64 bytes from 192.168.122.158: icmp_seq=1 ttl=64 time=0.140 ms
2022-03-10 02:56:36 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=2 ttl=64 time=0.141 ms
2022-03-10 02:56:37 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=3 ttl=64 time=0.081 ms
2022-03-10 02:56:38 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=4 ttl=64 time=0.160 ms
2022-03-10 02:56:39 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=5 ttl=64 time=0.083 ms

--- 192.168.122.158 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 77ms
rtt min/avg/max/mdev = 0.081/0.121/0.160/0.032 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 02:56:48 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.158
The authenticity of host '192.168.122.158 (192.168.122.158)' can't be established.
ECDSA key fingerprint is SHA256:uiL9MQ8j64kizIc2NkvuxVE93C2YBfGXw5p1QHW5dUA.
Are you sure you want to continue connecting (yes/no/[fingerprint])? 
2022-03-10 02:56:57 - sshInstance.py - [line:94] - root - INFO - yes
Warning: Permanently added '192.168.122.158' (ECDSA) to the list of known hosts.
root@192.168.122.158's password: 
2022-03-10 02:57:06 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 01:58:35 PM EST
  System load:  0.02               Processes:             134
  Usage of /:   37.2% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.158
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Tue Mar  1 02:51:01 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 02:57:15 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-10 02:57:21 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-10 02:57:27 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-10 02:57:32 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-10 02:57:38 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-10 02:57:41 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-10 02:57:47 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-10 02:57:56 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-10 02:58:05 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-10 02:58:14 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-10 02:58:23 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-10 02:58:32 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.158 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 02:58:40 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.16
PING 192.168.122.16 (192.168.122.16) 56(84) bytes of data.
64 bytes from 192.168.122.16: icmp_seq=1 ttl=64 time=0.197 ms
2022-03-10 02:58:41 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=2 ttl=64 time=0.156 ms
2022-03-10 02:58:42 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=3 ttl=64 time=0.101 ms
2022-03-10 02:58:43 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=4 ttl=64 time=0.084 ms
2022-03-10 02:58:44 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=5 ttl=64 time=0.092 ms

--- 192.168.122.16 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 86ms
rtt min/avg/max/mdev = 0.084/0.126/0.197/0.043 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 02:58:53 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.16
The authenticity of host '192.168.122.16 (192.168.122.16)' can't be established.
ECDSA key fingerprint is SHA256:uiL9MQ8j64kizIc2NkvuxVE93C2YBfGXw5p1QHW5dUA.
Are you sure you want to continue connecting (yes/no/[fingerprint])? 
2022-03-10 02:59:02 - sshInstance.py - [line:94] - root - INFO - yes
Warning: Permanently added '192.168.122.16' (ECDSA) to the list of known hosts.
root@192.168.122.16's password: 
2022-03-10 02:59:11 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 02:00:39 PM EST
  System load:  0.1                Processes:             137
  Usage of /:   37.2% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.16
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Tue Mar  1 02:51:57 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 02:59:20 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-10 02:59:26 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-10 02:59:32 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-10 02:59:37 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-10 02:59:43 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-10 02:59:46 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-10 02:59:52 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-10 03:00:01 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-10 03:00:10 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-10 03:00:19 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-10 03:00:28 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-10 03:00:37 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.16 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:00:40 - test_network_sriov_stress.py - [line:467] - root - INFO - Ping two vms IP from SUT:
2022-03-10 03:00:44 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.210 ms
2022-03-10 03:00:45 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.065 ms
2022-03-10 03:00:46 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.065 ms
2022-03-10 03:00:47 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.055 ms
2022-03-10 03:00:48 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.059 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 93ms
rtt min/avg/max/mdev = 0.055/0.090/0.210/0.060 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:00:55 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=5.10 ms
2022-03-10 03:00:56 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.118 ms
2022-03-10 03:00:57 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.067 ms
2022-03-10 03:00:58 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.066 ms
2022-03-10 03:00:59 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.107 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 86ms
rtt min/avg/max/mdev = 0.066/1.090/5.095/2.002 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:01:02 - test_network_sriov_stress.py - [line:475] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-10 03:01:06 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.158
PING 192.168.122.158 (192.168.122.158) 56(84) bytes of data.
64 bytes from 192.168.122.158: icmp_seq=1 ttl=64 time=0.166 ms
2022-03-10 03:01:07 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=2 ttl=64 time=0.088 ms
2022-03-10 03:01:08 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=3 ttl=64 time=0.078 ms
2022-03-10 03:01:09 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=4 ttl=64 time=0.089 ms
2022-03-10 03:01:10 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=5 ttl=64 time=0.148 ms

--- 192.168.122.158 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 79ms
rtt min/avg/max/mdev = 0.078/0.113/0.166/0.038 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:01:19 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.158
root@192.168.122.158's password: 
2022-03-10 03:01:28 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 02:02:56 PM EST
  System load:  0.0                Processes:             129
  Usage of /:   37.2% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.158
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Wed Mar  9 13:58:35 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 03:01:35 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=5.06 ms
2022-03-10 03:01:36 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.070 ms
2022-03-10 03:01:37 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.041 ms
2022-03-10 03:01:38 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.047 ms
2022-03-10 03:01:39 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.061 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4066ms
rtt min/avg/max/mdev = 0.041/1.055/5.060/2.002 ms
root@localhost:~# 
2022-03-10 03:02:19 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.158 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:02:22 - test_network_sriov_stress.py - [line:483] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-10 03:02:26 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
2022-03-10 03:02:29 - sshInstance.py - [line:71] - root - INFO - From 192.168.10.10 icmp_seq=1 Destination Host Unreachable
From 192.168.10.10 icmp_seq=2 Destination Host Unreachable
From 192.168.10.10 icmp_seq=3 Destination Host Unreachable
2022-03-10 03:02:32 - sshInstance.py - [line:71] - root - INFO - From 192.168.10.10 icmp_seq=4 Destination Host Unreachable
From 192.168.10.10 icmp_seq=5 Destination Host Unreachable

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 0 received, +5 errors, 100% packet loss, time 107ms
pipe 4
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:02:38 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-10 03:02:42 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:02:47 - test_network_sriov_stress.py - [line:262] - root - INFO - Successfully removed VFs from PF.
2022-03-10 03:02:50 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-10 03:02:51 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-10 03:02:56 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-10 03:02:59 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-10 03:02:59 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-10 03:03:04 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest1
2022-03-10 03:03:07 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-10 03:03:08 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-10 03:03:13 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-10 03:03:16 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-10 03:03:16 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-10 03:03:21 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest2
2022-03-10 03:03:21 - test_network_sriov_stress.py - [line:388] - root - INFO - 

********************************************************Stress Disable and Enable for 2 time when VF is Assigned VM guest.*************************************************************


2022-03-10 03:03:24 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -s | grep ens785f2 | awk '{print $1;}'
2022-03-10 03:03:24 - sshInstance.py - [line:50] - root - INFO - 0000:4b:00.2

2022-03-10 03:03:29 - test_network_sriov_stress.py - [line:108] - root - INFO - pf_id was found: 0000:4b:00.2

2022-03-10 03:03:29 - test_network_sriov_stress.py - [line:396] - root - INFO - format_pf_id is 0000\:4b\:00.2

2022-03-10 03:03:29 - test_network_sriov_stress.py - [line:399] - root - INFO - pf_id_key: 4b
2022-03-10 03:03:35 - sshInstance.py - [line:94] - root - INFO - ifconfig ens785f2 192.168.10.10
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:03:38 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens785f2
2022-03-10 03:03:41 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-10 03:03:42 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:03:47 - test_network_sriov_stress.py - [line:122] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-10 03:03:50 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-10 03:03:50 - sshInstance.py - [line:50] - root - INFO - 4b:11.0
4b:11.1
4b:11.2
4b:11.3

2022-03-10 03:03:55 - test_network_sriov_stress.py - [line:136] - root - INFO - Succssfully Gotten vf_id list:

2022-03-10 03:03:58 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-10 03:03:58 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:04:06 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.0
2022-03-10 03:04:07 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:04:12 - test_network_sriov_stress.py - [line:169] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-10 03:04:15 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.1
2022-03-10 03:04:16 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:04:21 - test_network_sriov_stress.py - [line:169] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-10 03:04:21 - test_network_sriov_stress.py - [line:420] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-10 03:04:24 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-10 03:04:24 - sshInstance.py - [line:50] - root - INFO - 0000:4b:11.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-03-10 03:04:29 - test_network_sriov_stress.py - [line:431] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-10 03:04:32 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-10 03:04:32 - sshInstance.py - [line:50] - root - INFO - yes

2022-03-10 03:04:37 - test_network_sriov_stress.py - [line:196] - root - INFO - Default network is active? yes

2022-03-10 03:04:41 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:04:44 - test_network_sriov_stress.py - [line:228] - root - ERROR - Failed to create a VM: virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:04:44 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: 
2022-03-10 03:04:48 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-10 03:04:51 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:04:54 - test_network_sriov_stress.py - [line:231] - root - INFO - Successfully created a VM: VMtest1
2022-03-10 03:04:58 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:05:01 - test_network_sriov_stress.py - [line:228] - root - ERROR - Failed to create a VM: virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:05:01 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: 
2022-03-10 03:05:05 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-10 03:05:08 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:05:11 - test_network_sriov_stress.py - [line:231] - root - INFO - Successfully created a VM: VMtest2
2022-03-10 03:05:14 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-10 03:05:14 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:05:19 - test_network_sriov_stress.py - [line:248] - root - ERROR - not found IP of VMtest1
2022-03-10 03:06:22 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-10 03:06:22 - sshInstance.py - [line:50] - root - INFO - 192.168.122.158/24

2022-03-10 03:06:27 - test_network_sriov_stress.py - [line:243] - root - INFO - Successfully got VMtest1 IP: 192.168.122.158/24

2022-03-10 03:06:30 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-10 03:06:30 - sshInstance.py - [line:50] - root - INFO - 192.168.122.16/24

2022-03-10 03:06:35 - test_network_sriov_stress.py - [line:243] - root - INFO - Successfully got VMtest2 IP: 192.168.122.16/24

2022-03-10 03:06:35 - test_network_sriov_stress.py - [line:448] - root - INFO - ip of VM1: 192.168.122.158
2022-03-10 03:06:35 - test_network_sriov_stress.py - [line:449] - root - INFO - ip of VM1: 192.168.122.16
2022-03-10 03:06:39 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.158
PING 192.168.122.158 (192.168.122.158) 56(84) bytes of data.
64 bytes from 192.168.122.158: icmp_seq=1 ttl=64 time=0.150 ms
2022-03-10 03:06:40 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=2 ttl=64 time=0.100 ms
2022-03-10 03:06:41 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=3 ttl=64 time=0.094 ms
2022-03-10 03:06:42 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=4 ttl=64 time=0.081 ms
2022-03-10 03:06:43 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=5 ttl=64 time=0.086 ms

--- 192.168.122.158 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 121ms
rtt min/avg/max/mdev = 0.081/0.102/0.150/0.025 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:06:52 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.158
root@192.168.122.158's password: 
2022-03-10 03:07:01 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 02:08:30 PM EST
  System load:  0.04               Processes:             136
  Usage of /:   37.2% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.158
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Wed Mar  9 14:02:56 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 03:07:10 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-10 03:07:16 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-10 03:07:22 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-10 03:07:27 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-10 03:07:33 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-10 03:07:36 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-10 03:07:42 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-10 03:07:51 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-10 03:08:00 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-10 03:08:10 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-10 03:08:19 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-10 03:08:28 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.158 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:08:35 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.16
PING 192.168.122.16 (192.168.122.16) 56(84) bytes of data.
64 bytes from 192.168.122.16: icmp_seq=1 ttl=64 time=0.165 ms
2022-03-10 03:08:36 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=2 ttl=64 time=0.103 ms
2022-03-10 03:08:37 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=3 ttl=64 time=0.163 ms
2022-03-10 03:08:38 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=4 ttl=64 time=0.081 ms
2022-03-10 03:08:39 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=5 ttl=64 time=0.101 ms

--- 192.168.122.16 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 114ms
rtt min/avg/max/mdev = 0.081/0.122/0.165/0.036 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:08:48 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.16
root@192.168.122.16's password: 
2022-03-10 03:08:57 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 02:10:25 PM EST
  System load:  0.0                Processes:             136
  Usage of /:   37.2% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.16
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Wed Mar  9 14:00:39 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 03:09:06 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-10 03:09:12 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-10 03:09:18 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-10 03:09:23 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-10 03:09:29 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-10 03:09:32 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-10 03:09:38 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-10 03:09:47 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-10 03:09:56 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-10 03:10:05 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-10 03:10:14 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-10 03:10:23 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.16 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:10:26 - test_network_sriov_stress.py - [line:467] - root - INFO - Ping two vms IP from SUT:
2022-03-10 03:10:30 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=5.26 ms
2022-03-10 03:10:31 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.067 ms
2022-03-10 03:10:32 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.077 ms
2022-03-10 03:10:33 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.043 ms
2022-03-10 03:10:34 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.043 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 109ms
rtt min/avg/max/mdev = 0.043/1.098/5.261/2.081 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:10:41 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.168 ms
2022-03-10 03:10:42 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.054 ms
2022-03-10 03:10:43 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.056 ms
2022-03-10 03:10:44 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.050 ms
2022-03-10 03:10:45 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.046 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 102ms
rtt min/avg/max/mdev = 0.046/0.074/0.168/0.047 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:10:48 - test_network_sriov_stress.py - [line:475] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-10 03:10:52 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.158
PING 192.168.122.158 (192.168.122.158) 56(84) bytes of data.
64 bytes from 192.168.122.158: icmp_seq=1 ttl=64 time=0.181 ms
2022-03-10 03:10:53 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=2 ttl=64 time=0.164 ms
2022-03-10 03:10:54 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=3 ttl=64 time=0.114 ms
2022-03-10 03:10:55 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=4 ttl=64 time=0.096 ms
2022-03-10 03:10:56 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=5 ttl=64 time=0.093 ms

--- 192.168.122.158 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 94ms
rtt min/avg/max/mdev = 0.093/0.129/0.181/0.038 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:11:05 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.158
root@192.168.122.158's password: 
2022-03-10 03:11:14 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 02:12:42 PM EST
  System load:  0.0                Processes:             128
  Usage of /:   37.2% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.158
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Wed Mar  9 14:08:30 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 03:11:21 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.212 ms
2022-03-10 03:11:22 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.050 ms
2022-03-10 03:11:23 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.081 ms
2022-03-10 03:11:24 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.069 ms
2022-03-10 03:11:25 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.065 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4097ms
rtt min/avg/max/mdev = 0.050/0.095/0.212/0.059 ms
root@localhost:~# 
2022-03-10 03:12:05 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.158 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:12:08 - test_network_sriov_stress.py - [line:483] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-10 03:12:12 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
2022-03-10 03:12:15 - sshInstance.py - [line:71] - root - INFO - From 192.168.10.10 icmp_seq=1 Destination Host Unreachable
From 192.168.10.10 icmp_seq=2 Destination Host Unreachable
From 192.168.10.10 icmp_seq=3 Destination Host Unreachable
2022-03-10 03:12:18 - sshInstance.py - [line:71] - root - INFO - From 192.168.10.10 icmp_seq=4 Destination Host Unreachable
From 192.168.10.10 icmp_seq=5 Destination Host Unreachable

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 0 received, +5 errors, 100% packet loss, time 122ms
pipe 4
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:12:24 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-10 03:12:28 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:12:33 - test_network_sriov_stress.py - [line:262] - root - INFO - Successfully removed VFs from PF.
2022-03-10 03:12:36 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-10 03:12:37 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-10 03:12:42 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-10 03:12:45 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-10 03:12:45 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-10 03:12:50 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest1
2022-03-10 03:12:53 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-10 03:12:54 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-10 03:12:59 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-10 03:13:02 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-10 03:13:02 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-10 03:13:07 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest2
2022-03-10 03:13:07 - test_network_sriov_stress.py - [line:388] - root - INFO - 

********************************************************Stress Disable and Enable for 3 time when VF is Assigned VM guest.*************************************************************


2022-03-10 03:13:10 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -s | grep ens785f2 | awk '{print $1;}'
2022-03-10 03:13:11 - sshInstance.py - [line:50] - root - INFO - 0000:4b:00.2

2022-03-10 03:13:16 - test_network_sriov_stress.py - [line:108] - root - INFO - pf_id was found: 0000:4b:00.2

2022-03-10 03:13:16 - test_network_sriov_stress.py - [line:396] - root - INFO - format_pf_id is 0000\:4b\:00.2

2022-03-10 03:13:16 - test_network_sriov_stress.py - [line:399] - root - INFO - pf_id_key: 4b
2022-03-10 03:13:22 - sshInstance.py - [line:94] - root - INFO - ifconfig ens785f2 192.168.10.10
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:13:25 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens785f2
2022-03-10 03:13:28 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-10 03:13:28 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:13:33 - test_network_sriov_stress.py - [line:122] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-10 03:13:36 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-10 03:13:36 - sshInstance.py - [line:50] - root - INFO - 4b:11.0
4b:11.1
4b:11.2
4b:11.3

2022-03-10 03:13:41 - test_network_sriov_stress.py - [line:136] - root - INFO - Succssfully Gotten vf_id list:

2022-03-10 03:13:44 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-10 03:13:44 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:13:52 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.0
2022-03-10 03:13:53 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:13:58 - test_network_sriov_stress.py - [line:169] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-10 03:14:01 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.1
2022-03-10 03:14:02 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:14:07 - test_network_sriov_stress.py - [line:169] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-10 03:14:07 - test_network_sriov_stress.py - [line:420] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-10 03:14:10 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-10 03:14:10 - sshInstance.py - [line:50] - root - INFO - 0000:4b:11.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-03-10 03:14:15 - test_network_sriov_stress.py - [line:431] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-10 03:14:18 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-10 03:14:18 - sshInstance.py - [line:50] - root - INFO - yes

2022-03-10 03:14:23 - test_network_sriov_stress.py - [line:196] - root - INFO - Default network is active? yes

2022-03-10 03:14:27 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:14:30 - test_network_sriov_stress.py - [line:228] - root - ERROR - Failed to create a VM: virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:14:30 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: 
2022-03-10 03:14:34 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-10 03:14:37 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:14:40 - test_network_sriov_stress.py - [line:231] - root - INFO - Successfully created a VM: VMtest1
2022-03-10 03:14:44 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:14:47 - test_network_sriov_stress.py - [line:228] - root - ERROR - Failed to create a VM: virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:14:47 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: 
2022-03-10 03:14:51 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-10 03:14:54 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:14:57 - test_network_sriov_stress.py - [line:231] - root - INFO - Successfully created a VM: VMtest2
2022-03-10 03:15:00 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-10 03:15:00 - sshInstance.py - [line:50] - root - INFO - 192.168.122.158/24

2022-03-10 03:15:05 - test_network_sriov_stress.py - [line:243] - root - INFO - Successfully got VMtest1 IP: 192.168.122.158/24

2022-03-10 03:15:08 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-10 03:15:08 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:15:13 - test_network_sriov_stress.py - [line:248] - root - ERROR - not found IP of VMtest2
2022-03-10 03:16:17 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-10 03:16:17 - sshInstance.py - [line:50] - root - INFO - 192.168.122.16/24

2022-03-10 03:16:22 - test_network_sriov_stress.py - [line:243] - root - INFO - Successfully got VMtest2 IP: 192.168.122.16/24

2022-03-10 03:16:22 - test_network_sriov_stress.py - [line:448] - root - INFO - ip of VM1: 192.168.122.158
2022-03-10 03:16:22 - test_network_sriov_stress.py - [line:449] - root - INFO - ip of VM1: 192.168.122.16
2022-03-10 03:16:26 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.158
PING 192.168.122.158 (192.168.122.158) 56(84) bytes of data.
64 bytes from 192.168.122.158: icmp_seq=1 ttl=64 time=0.129 ms
2022-03-10 03:16:27 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=2 ttl=64 time=0.125 ms
2022-03-10 03:16:28 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=3 ttl=64 time=0.096 ms
2022-03-10 03:16:29 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=4 ttl=64 time=0.080 ms
2022-03-10 03:16:30 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=5 ttl=64 time=0.082 ms

--- 192.168.122.158 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 91ms
rtt min/avg/max/mdev = 0.080/0.102/0.129/0.022 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:16:39 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.158
root@192.168.122.158's password: 
2022-03-10 03:16:48 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 02:18:16 PM EST
  System load:  0.29               Processes:             147
  Usage of /:   37.3% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.158
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Wed Mar  9 14:12:43 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 03:16:57 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-10 03:17:03 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-10 03:17:09 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-10 03:17:14 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-10 03:17:20 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-10 03:17:23 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-10 03:17:29 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-10 03:17:38 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-10 03:17:47 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-10 03:17:56 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-10 03:18:05 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-10 03:18:14 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.158 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:18:21 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.16
PING 192.168.122.16 (192.168.122.16) 56(84) bytes of data.
64 bytes from 192.168.122.16: icmp_seq=1 ttl=64 time=0.169 ms
2022-03-10 03:18:22 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=2 ttl=64 time=0.165 ms
2022-03-10 03:18:23 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=3 ttl=64 time=0.120 ms
2022-03-10 03:18:24 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=4 ttl=64 time=0.094 ms
2022-03-10 03:18:25 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=5 ttl=64 time=0.138 ms

--- 192.168.122.16 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 83ms
rtt min/avg/max/mdev = 0.094/0.137/0.169/0.029 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:18:34 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.16
root@192.168.122.16's password: 
2022-03-10 03:18:43 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 02:20:11 PM EST
  System load:  0.0                Processes:             139
  Usage of /:   37.3% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.16
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Wed Mar  9 14:10:26 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 03:18:52 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-10 03:18:58 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-10 03:19:04 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-10 03:19:09 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-10 03:19:15 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-10 03:19:18 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-10 03:19:24 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-10 03:19:33 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-10 03:19:42 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-10 03:19:51 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-10 03:20:00 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 346 (346.0 B)
        RX errors 0  dropped 1  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-10 03:20:09 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.16 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:20:12 - test_network_sriov_stress.py - [line:467] - root - INFO - Ping two vms IP from SUT:
2022-03-10 03:20:16 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.204 ms
2022-03-10 03:20:17 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.053 ms
2022-03-10 03:20:18 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.068 ms
2022-03-10 03:20:19 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.046 ms
2022-03-10 03:20:20 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.058 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 139ms
rtt min/avg/max/mdev = 0.046/0.085/0.204/0.060 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:20:27 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=4.07 ms
2022-03-10 03:20:28 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.088 ms
2022-03-10 03:20:29 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.047 ms
2022-03-10 03:20:30 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.039 ms
2022-03-10 03:20:31 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.055 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 67ms
rtt min/avg/max/mdev = 0.039/0.859/4.069/1.605 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:20:34 - test_network_sriov_stress.py - [line:475] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-10 03:20:38 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.158
PING 192.168.122.158 (192.168.122.158) 56(84) bytes of data.
64 bytes from 192.168.122.158: icmp_seq=1 ttl=64 time=0.161 ms
2022-03-10 03:20:39 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=2 ttl=64 time=0.155 ms
2022-03-10 03:20:40 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=3 ttl=64 time=0.098 ms
2022-03-10 03:20:41 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=4 ttl=64 time=0.095 ms
2022-03-10 03:20:42 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=5 ttl=64 time=0.102 ms

--- 192.168.122.158 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 123ms
rtt min/avg/max/mdev = 0.095/0.122/0.161/0.030 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:20:51 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.158
root@192.168.122.158's password: 
2022-03-10 03:21:00 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 02:22:28 PM EST
  System load:  0.0                Processes:             130
  Usage of /:   37.3% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.158
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Wed Mar  9 14:18:16 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 03:21:07 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=4.78 ms
2022-03-10 03:21:08 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=3.76 ms
2022-03-10 03:21:09 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.051 ms
2022-03-10 03:21:10 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.056 ms
2022-03-10 03:21:11 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.045 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4048ms
rtt min/avg/max/mdev = 0.045/1.738/4.777/2.091 ms
root@localhost:~# 
2022-03-10 03:21:51 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.158 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:21:54 - test_network_sriov_stress.py - [line:483] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-10 03:21:58 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
2022-03-10 03:22:01 - sshInstance.py - [line:71] - root - INFO - From 192.168.10.10 icmp_seq=1 Destination Host Unreachable
From 192.168.10.10 icmp_seq=2 Destination Host Unreachable
From 192.168.10.10 icmp_seq=3 Destination Host Unreachable
2022-03-10 03:22:04 - sshInstance.py - [line:71] - root - INFO - From 192.168.10.10 icmp_seq=4 Destination Host Unreachable
From 192.168.10.10 icmp_seq=5 Destination Host Unreachable

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 0 received, +5 errors, 100% packet loss, time 87ms
pipe 4
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:22:10 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-10 03:22:14 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:22:19 - test_network_sriov_stress.py - [line:262] - root - INFO - Successfully removed VFs from PF.
2022-03-10 03:22:22 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-10 03:22:23 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-10 03:22:28 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-10 03:22:31 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-10 03:22:31 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-10 03:22:36 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest1
2022-03-10 03:22:39 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-10 03:22:40 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-10 03:22:45 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-10 03:22:48 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-10 03:22:48 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-10 03:22:53 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest2
2022-03-10 03:22:53 - test_network_sriov_stress.py - [line:388] - root - INFO - 

********************************************************Stress Disable and Enable for 4 time when VF is Assigned VM guest.*************************************************************


2022-03-10 03:22:56 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -s | grep ens785f2 | awk '{print $1;}'
2022-03-10 03:22:57 - sshInstance.py - [line:50] - root - INFO - 0000:4b:00.2

2022-03-10 03:23:02 - test_network_sriov_stress.py - [line:108] - root - INFO - pf_id was found: 0000:4b:00.2

2022-03-10 03:23:02 - test_network_sriov_stress.py - [line:396] - root - INFO - format_pf_id is 0000\:4b\:00.2

2022-03-10 03:23:02 - test_network_sriov_stress.py - [line:399] - root - INFO - pf_id_key: 4b
2022-03-10 03:23:08 - sshInstance.py - [line:94] - root - INFO - ifconfig ens785f2 192.168.10.10
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:23:11 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens785f2
2022-03-10 03:23:14 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-10 03:23:14 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:23:19 - test_network_sriov_stress.py - [line:122] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-10 03:23:22 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-10 03:23:22 - sshInstance.py - [line:50] - root - INFO - 4b:11.0
4b:11.1
4b:11.2
4b:11.3

2022-03-10 03:23:27 - test_network_sriov_stress.py - [line:136] - root - INFO - Succssfully Gotten vf_id list:

2022-03-10 03:23:30 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-10 03:23:30 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:23:38 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.0
2022-03-10 03:23:39 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:23:44 - test_network_sriov_stress.py - [line:169] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-10 03:23:47 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.1
2022-03-10 03:23:48 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:23:53 - test_network_sriov_stress.py - [line:169] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-10 03:23:53 - test_network_sriov_stress.py - [line:420] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-10 03:23:56 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-10 03:23:56 - sshInstance.py - [line:50] - root - INFO - 0000:4b:11.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-03-10 03:24:01 - test_network_sriov_stress.py - [line:431] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-10 03:24:04 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-10 03:24:04 - sshInstance.py - [line:50] - root - INFO - yes

2022-03-10 03:24:09 - test_network_sriov_stress.py - [line:196] - root - INFO - Default network is active? yes

2022-03-10 03:24:13 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:24:16 - test_network_sriov_stress.py - [line:228] - root - ERROR - Failed to create a VM: virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:24:16 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: 
2022-03-10 03:24:20 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-10 03:24:23 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:24:26 - test_network_sriov_stress.py - [line:231] - root - INFO - Successfully created a VM: VMtest1
2022-03-10 03:24:30 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:24:33 - test_network_sriov_stress.py - [line:228] - root - ERROR - Failed to create a VM: virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:24:33 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: 
2022-03-10 03:24:37 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-10 03:24:40 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:24:43 - test_network_sriov_stress.py - [line:231] - root - INFO - Successfully created a VM: VMtest2
2022-03-10 03:24:46 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-10 03:24:47 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:24:52 - test_network_sriov_stress.py - [line:248] - root - ERROR - not found IP of VMtest1
2022-03-10 03:25:55 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-10 03:25:55 - sshInstance.py - [line:50] - root - INFO - 192.168.122.158/24

2022-03-10 03:26:00 - test_network_sriov_stress.py - [line:243] - root - INFO - Successfully got VMtest1 IP: 192.168.122.158/24

2022-03-10 03:26:03 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-10 03:26:03 - sshInstance.py - [line:50] - root - INFO - 192.168.122.16/24

2022-03-10 03:26:08 - test_network_sriov_stress.py - [line:243] - root - INFO - Successfully got VMtest2 IP: 192.168.122.16/24

2022-03-10 03:26:08 - test_network_sriov_stress.py - [line:448] - root - INFO - ip of VM1: 192.168.122.158
2022-03-10 03:26:08 - test_network_sriov_stress.py - [line:449] - root - INFO - ip of VM1: 192.168.122.16
2022-03-10 03:26:12 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.158
PING 192.168.122.158 (192.168.122.158) 56(84) bytes of data.
64 bytes from 192.168.122.158: icmp_seq=1 ttl=64 time=0.146 ms
2022-03-10 03:26:13 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=2 ttl=64 time=0.160 ms
2022-03-10 03:26:14 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=3 ttl=64 time=0.100 ms
2022-03-10 03:26:15 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=4 ttl=64 time=0.105 ms
2022-03-10 03:26:16 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=5 ttl=64 time=0.093 ms

--- 192.168.122.158 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 94ms
rtt min/avg/max/mdev = 0.093/0.120/0.160/0.030 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:26:25 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.158
root@192.168.122.158's password: 
2022-03-10 03:26:34 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 02:28:03 PM EST
  System load:  0.0                Processes:             144
  Usage of /:   37.3% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.158
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Wed Mar  9 14:22:28 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 03:26:43 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-10 03:26:49 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-10 03:26:55 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-10 03:27:00 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-10 03:27:06 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-10 03:27:09 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-10 03:27:15 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-10 03:27:24 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-10 03:27:33 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-10 03:27:42 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-10 03:27:51 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-10 03:28:00 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.158 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:28:07 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.16
PING 192.168.122.16 (192.168.122.16) 56(84) bytes of data.
64 bytes from 192.168.122.16: icmp_seq=1 ttl=64 time=0.139 ms
2022-03-10 03:28:08 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=2 ttl=64 time=0.145 ms
2022-03-10 03:28:09 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=3 ttl=64 time=0.137 ms
2022-03-10 03:28:10 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=4 ttl=64 time=0.136 ms
2022-03-10 03:28:11 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=5 ttl=64 time=0.101 ms

--- 192.168.122.16 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 86ms
rtt min/avg/max/mdev = 0.101/0.131/0.145/0.020 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:28:20 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.16
root@192.168.122.16's password: 
2022-03-10 03:28:29 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 02:29:58 PM EST
  System load:  0.01               Processes:             139
  Usage of /:   37.3% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.16
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Wed Mar  9 14:20:11 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 03:28:38 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-10 03:28:44 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-10 03:28:50 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-10 03:28:55 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-10 03:29:01 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-10 03:29:04 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-10 03:29:10 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-10 03:29:19 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-10 03:29:28 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-10 03:29:37 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-10 03:29:46 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 2  bytes 692 (692.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-10 03:29:55 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.16 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:29:58 - test_network_sriov_stress.py - [line:467] - root - INFO - Ping two vms IP from SUT:
2022-03-10 03:30:02 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=4.51 ms
2022-03-10 03:30:03 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.079 ms
2022-03-10 03:30:04 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.102 ms
2022-03-10 03:30:05 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.085 ms
2022-03-10 03:30:06 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.060 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 81ms
rtt min/avg/max/mdev = 0.060/0.966/4.508/1.771 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:30:13 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.210 ms
2022-03-10 03:30:14 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.103 ms
2022-03-10 03:30:15 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.043 ms
2022-03-10 03:30:16 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.039 ms
2022-03-10 03:30:17 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.042 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 137ms
rtt min/avg/max/mdev = 0.039/0.087/0.210/0.066 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:30:20 - test_network_sriov_stress.py - [line:475] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-10 03:30:24 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.158
PING 192.168.122.158 (192.168.122.158) 56(84) bytes of data.
64 bytes from 192.168.122.158: icmp_seq=1 ttl=64 time=0.136 ms
2022-03-10 03:30:25 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=2 ttl=64 time=0.164 ms
2022-03-10 03:30:26 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=3 ttl=64 time=0.144 ms
2022-03-10 03:30:27 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=4 ttl=64 time=0.095 ms
2022-03-10 03:30:28 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=5 ttl=64 time=0.113 ms

--- 192.168.122.158 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 129ms
rtt min/avg/max/mdev = 0.095/0.130/0.164/0.026 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:30:37 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.158
root@192.168.122.158's password: 
2022-03-10 03:30:46 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 02:32:15 PM EST
  System load:  0.0                Processes:             130
  Usage of /:   37.3% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.158
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Wed Mar  9 14:28:04 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 03:30:53 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.187 ms
2022-03-10 03:30:54 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.080 ms
2022-03-10 03:30:55 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.105 ms
2022-03-10 03:30:56 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.070 ms
2022-03-10 03:30:57 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.084 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4099ms
rtt min/avg/max/mdev = 0.070/0.105/0.187/0.042 ms
root@localhost:~# 
2022-03-10 03:31:37 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.158 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:31:40 - test_network_sriov_stress.py - [line:483] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-10 03:31:44 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
2022-03-10 03:31:47 - sshInstance.py - [line:71] - root - INFO - From 192.168.10.10 icmp_seq=1 Destination Host Unreachable
2022-03-10 03:31:50 - sshInstance.py - [line:71] - root - INFO - From 192.168.10.10 icmp_seq=5 Destination Host Unreachable
2022-03-10 03:31:58 - sshInstance.py - [line:71] - root - INFO - 
--- 192.168.10.11 ping statistics ---
5 packets transmitted, 0 received, +2 errors, 100% packet loss, time 95ms
pipe 4
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:32:04 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-10 03:32:08 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:32:13 - test_network_sriov_stress.py - [line:262] - root - INFO - Successfully removed VFs from PF.
2022-03-10 03:32:16 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-10 03:32:17 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-10 03:32:22 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-10 03:32:25 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-10 03:32:25 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-10 03:32:30 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest1
2022-03-10 03:32:33 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-10 03:32:34 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-10 03:32:39 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-10 03:32:42 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-10 03:32:42 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-10 03:32:47 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest2
2022-03-10 03:32:47 - test_network_sriov_stress.py - [line:388] - root - INFO - 

********************************************************Stress Disable and Enable for 5 time when VF is Assigned VM guest.*************************************************************


2022-03-10 03:32:50 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -s | grep ens785f2 | awk '{print $1;}'
2022-03-10 03:32:51 - sshInstance.py - [line:50] - root - INFO - 0000:4b:00.2

2022-03-10 03:32:56 - test_network_sriov_stress.py - [line:108] - root - INFO - pf_id was found: 0000:4b:00.2

2022-03-10 03:32:56 - test_network_sriov_stress.py - [line:396] - root - INFO - format_pf_id is 0000\:4b\:00.2

2022-03-10 03:32:56 - test_network_sriov_stress.py - [line:399] - root - INFO - pf_id_key: 4b
2022-03-10 03:33:02 - sshInstance.py - [line:94] - root - INFO - ifconfig ens785f2 192.168.10.10
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:33:05 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens785f2
2022-03-10 03:33:08 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-10 03:33:08 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:33:13 - test_network_sriov_stress.py - [line:122] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-10 03:33:16 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-10 03:33:16 - sshInstance.py - [line:50] - root - INFO - 4b:11.0
4b:11.1
4b:11.2
4b:11.3

2022-03-10 03:33:21 - test_network_sriov_stress.py - [line:136] - root - INFO - Succssfully Gotten vf_id list:

2022-03-10 03:33:24 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-10 03:33:24 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:33:32 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.0
2022-03-10 03:33:33 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:33:38 - test_network_sriov_stress.py - [line:169] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-10 03:33:41 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.1
2022-03-10 03:33:42 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:33:47 - test_network_sriov_stress.py - [line:169] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-10 03:33:47 - test_network_sriov_stress.py - [line:420] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-10 03:33:50 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-10 03:33:51 - sshInstance.py - [line:50] - root - INFO - 0000:4b:11.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-03-10 03:33:56 - test_network_sriov_stress.py - [line:431] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-10 03:33:59 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-10 03:33:59 - sshInstance.py - [line:50] - root - INFO - yes

2022-03-10 03:34:04 - test_network_sriov_stress.py - [line:196] - root - INFO - Default network is active? yes

2022-03-10 03:34:08 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:34:11 - test_network_sriov_stress.py - [line:228] - root - ERROR - Failed to create a VM: virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:34:11 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: 
2022-03-10 03:34:15 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-10 03:34:18 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:34:21 - test_network_sriov_stress.py - [line:231] - root - INFO - Successfully created a VM: VMtest1
2022-03-10 03:34:25 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:34:28 - test_network_sriov_stress.py - [line:228] - root - ERROR - Failed to create a VM: virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:34:28 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: 
2022-03-10 03:34:32 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-10 03:34:35 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:34:38 - test_network_sriov_stress.py - [line:231] - root - INFO - Successfully created a VM: VMtest2
2022-03-10 03:34:41 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-10 03:34:41 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:34:46 - test_network_sriov_stress.py - [line:248] - root - ERROR - not found IP of VMtest1
2022-03-10 03:35:49 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-10 03:35:49 - sshInstance.py - [line:50] - root - INFO - 192.168.122.158/24

2022-03-10 03:35:54 - test_network_sriov_stress.py - [line:243] - root - INFO - Successfully got VMtest1 IP: 192.168.122.158/24

2022-03-10 03:35:57 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-10 03:35:57 - sshInstance.py - [line:50] - root - INFO - 192.168.122.16/24

2022-03-10 03:36:02 - test_network_sriov_stress.py - [line:243] - root - INFO - Successfully got VMtest2 IP: 192.168.122.16/24

2022-03-10 03:36:02 - test_network_sriov_stress.py - [line:448] - root - INFO - ip of VM1: 192.168.122.158
2022-03-10 03:36:02 - test_network_sriov_stress.py - [line:449] - root - INFO - ip of VM1: 192.168.122.16
2022-03-10 03:36:06 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.158
PING 192.168.122.158 (192.168.122.158) 56(84) bytes of data.
64 bytes from 192.168.122.158: icmp_seq=1 ttl=64 time=0.171 ms
2022-03-10 03:36:07 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=2 ttl=64 time=0.104 ms
2022-03-10 03:36:08 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=3 ttl=64 time=0.110 ms
2022-03-10 03:36:09 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=4 ttl=64 time=0.129 ms
2022-03-10 03:36:10 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=5 ttl=64 time=0.104 ms

--- 192.168.122.158 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 102ms
rtt min/avg/max/mdev = 0.104/0.123/0.171/0.028 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:36:19 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.158
root@192.168.122.158's password: 
2022-03-10 03:36:28 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 02:37:57 PM EST
  System load:  0.01               Processes:             142
  Usage of /:   37.3% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.158
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Wed Mar  9 14:32:16 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 03:36:37 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-10 03:36:43 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-10 03:36:49 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-10 03:36:54 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-10 03:37:00 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-10 03:37:03 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-10 03:37:09 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-10 03:37:18 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-10 03:37:27 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-10 03:37:36 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-10 03:37:45 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-10 03:37:54 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.158 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:38:01 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.16
PING 192.168.122.16 (192.168.122.16) 56(84) bytes of data.
64 bytes from 192.168.122.16: icmp_seq=1 ttl=64 time=0.124 ms
2022-03-10 03:38:02 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=2 ttl=64 time=0.085 ms
2022-03-10 03:38:03 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=3 ttl=64 time=0.117 ms
2022-03-10 03:38:04 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=4 ttl=64 time=0.122 ms
2022-03-10 03:38:05 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.16: icmp_seq=5 ttl=64 time=0.071 ms

--- 192.168.122.16 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 94ms
rtt min/avg/max/mdev = 0.071/0.103/0.124/0.025 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:38:14 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.16
root@192.168.122.16's password: 
2022-03-10 03:38:23 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 02:39:52 PM EST
  System load:  0.0                Processes:             137
  Usage of /:   37.4% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.16
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Wed Mar  9 14:29:59 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 03:38:32 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-10 03:38:38 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-10 03:38:44 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-10 03:38:49 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-10 03:38:55 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-10 03:38:58 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-10 03:39:04 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-10 03:39:13 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-10 03:39:22 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-10 03:39:31 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-10 03:39:40 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-10 03:39:49 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.16 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:39:52 - test_network_sriov_stress.py - [line:467] - root - INFO - Ping two vms IP from SUT:
2022-03-10 03:39:56 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=3.52 ms
2022-03-10 03:39:57 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.040 ms
2022-03-10 03:39:58 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.075 ms
2022-03-10 03:39:59 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.044 ms
2022-03-10 03:40:00 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.074 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 89ms
rtt min/avg/max/mdev = 0.040/0.750/3.517/1.383 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:40:07 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.193 ms
2022-03-10 03:40:08 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.080 ms
2022-03-10 03:40:09 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.045 ms
2022-03-10 03:40:10 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.048 ms
2022-03-10 03:40:11 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.040 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 82ms
rtt min/avg/max/mdev = 0.040/0.081/0.193/0.057 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:40:14 - test_network_sriov_stress.py - [line:475] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-10 03:40:18 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.158
PING 192.168.122.158 (192.168.122.158) 56(84) bytes of data.
64 bytes from 192.168.122.158: icmp_seq=1 ttl=64 time=0.137 ms
2022-03-10 03:40:19 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=2 ttl=64 time=0.126 ms
2022-03-10 03:40:20 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=3 ttl=64 time=0.130 ms
2022-03-10 03:40:21 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=4 ttl=64 time=0.095 ms
2022-03-10 03:40:22 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.158: icmp_seq=5 ttl=64 time=0.100 ms

--- 192.168.122.158 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 138ms
rtt min/avg/max/mdev = 0.095/0.117/0.137/0.020 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:40:31 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.158
root@192.168.122.158's password: 
2022-03-10 03:40:40 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Wed 09 Mar 2022 02:42:10 PM EST
  System load:  0.0                Processes:             127
  Usage of /:   37.3% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.158
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Wed Mar  9 14:37:58 2022 from 192.168.122.1
root@localhost:~# 
2022-03-10 03:40:47 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=4.35 ms
2022-03-10 03:40:48 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.073 ms
2022-03-10 03:40:49 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.051 ms
2022-03-10 03:40:50 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.062 ms
2022-03-10 03:40:51 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.051 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4064ms
rtt min/avg/max/mdev = 0.051/0.916/4.346/1.714 ms
root@localhost:~# 
2022-03-10 03:41:31 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.158 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:41:34 - test_network_sriov_stress.py - [line:483] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-10 03:41:38 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
2022-03-10 03:41:41 - sshInstance.py - [line:71] - root - INFO - From 192.168.10.10 icmp_seq=1 Destination Host Unreachable
From 192.168.10.10 icmp_seq=2 Destination Host Unreachable
From 192.168.10.10 icmp_seq=3 Destination Host Unreachable
2022-03-10 03:41:44 - sshInstance.py - [line:71] - root - INFO - From 192.168.10.10 icmp_seq=4 Destination Host Unreachable
From 192.168.10.10 icmp_seq=5 Destination Host Unreachable

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 0 received, +5 errors, 100% packet loss, time 103ms
pipe 4
[root@npx-cyp7-rh82 ~]# 
2022-03-10 03:41:50 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-10 03:41:54 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:41:59 - test_network_sriov_stress.py - [line:262] - root - INFO - Successfully removed VFs from PF.
2022-03-10 03:42:02 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-10 03:42:03 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-10 03:42:08 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-10 03:42:11 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-10 03:42:11 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-10 03:42:16 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest1
2022-03-10 03:42:19 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-10 03:42:20 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-10 03:42:25 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-10 03:42:28 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-10 03:42:29 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-10 03:42:34 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest2
.2022-03-10 03:42:37 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-10 03:42:37 - sshInstance.py - [line:50] - root - INFO - error: failed to get domain 'VMtest2'


2022-03-10 03:42:42 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-10 03:42:45 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-10 03:42:45 - sshInstance.py - [line:50] - root - INFO - error: failed to get domain 'VMtest2'


2022-03-10 03:42:50 - test_network_sriov_stress.py - [line:290] - root - ERROR - Failed to removed the VM VMtest2
2022-03-10 03:42:53 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-10 03:42:53 - sshInstance.py - [line:50] - root - INFO - error: failed to get domain 'VMtest1'


2022-03-10 03:42:58 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-10 03:43:01 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-10 03:43:01 - sshInstance.py - [line:50] - root - INFO - error: failed to get domain 'VMtest1'


2022-03-10 03:43:06 - test_network_sriov_stress.py - [line:290] - root - ERROR - Failed to removed the VM VMtest1
2022-03-10 03:43:09 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-10 03:43:09 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:43:17 - sshInstance.py - [line:46] - root - INFO - ifconfig ens785f2 0
2022-03-10 03:43:17 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:43:25 - sshInstance.py - [line:46] - root - INFO - dmesg > /tmp/dmesg.log
2022-03-10 03:43:25 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:43:30 - test_network_sriov_stress.py - [line:79] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-03-10 03:43:30 - test_network_sriov_stress.py - [line:81] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-03-10/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm2.img-uvm1.img]_02_52_33
2022-03-10 03:43:33 - sshInstance.py - [line:46] - root - INFO - rm -f /tmp/dmesg.log
2022-03-10 03:43:33 - sshInstance.py - [line:50] - root - INFO - 
2022-03-10 03:43:38 - test_network_sriov_stress.py - [line:83] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-03-10 03:43:38 - sshInstance.py - [line:39] - root - INFO - closing ssh client on SUT
2022-03-10 03:43:38 - NicTestFixture.py - [line:32] - root - INFO - ============================ end of run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm2.img-uvm1.img] case ============================
                  [100%]

=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:62
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:62: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:101
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:101: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:71
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:71: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:110
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:110: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:381
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:381: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:495
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:495: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:791
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:791: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:78
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:78: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:101
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:101: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:122
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:122: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:189
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:189: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:213
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:213: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html -
========== 1 passed, 15 deselected, 12 warnings in 3064.98s (0:51:04) ==========
[8mha:////4JadWPtzQEFx9CzixTlssp2D6uuorSv3wnTa6h1wuGrmAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiISEKRJXWcsMLTGyME+vO2A5JxYv4Gn8gEImKLVY728zzBcsU4cjRYmuoc5Qah8H3dlo4cOwungds+YwNU2JvUJpBsjb1jJKzgTlFCQsBK0ON5+TIZliLVt1V5RXZ6pTj9B0ElE7f4AGFmMxZxfyFMcLm2uvE9PP8FcAYQoZyu//U7g3QKkBhvgAAAA==[0m[Pipeline] }
[8mha:////4Goa9rseL6gCK8+j51lIjOVJAeE64BF71YmRtb4KtKxFAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiIQokRJXWcsMLTGyME+vO2A5JxYv4Gn8gEImKLVY728zzBcsU4cjRYmuoc5Qah8H3dlo4cOwungds+YwNU2JvUJpBsjb1jJKzgTlFCQsBK0ON5+TIZliLVt1V5RXZ6pTj9B0ElE7f4AGFmMxZxfyFMcLm2uvE9PP8FcAYQoZyu//U7g2TqBHnvgAAAA==[0m[Pipeline] // stage
[8mha:////4AK444efjkykafC1HGbgxn6nPSQkL/HNauaXETS7R91oAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjIRqEqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A+OvzUfBAAAA[0m[Pipeline] stage
[8mha:////4EqpMgzslAWPDRl1bp4feAKRfPoiBNazp8uDBURuqJGsAAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycogUwYRQJ9aoCycITQhJo/9LkpJOnIircQcClZjwYNmWrPd8QRU8NOQ1swp7g6EzbHCjzokl8v3FUWKWzqwjDOQUa1VqSarjXFuKCmYVJSw4LBV2joJBHWHFrbiL2gnU9Sn6vB04lEbe4AEFz+QofPyWycP6OspA+OP8BcA0RCi3+2y7zedZvQH4j6CMwgAAAA==[0m[Pipeline] { (collect logs)
[8mha:////4EXomUzh+l1yxKUNNmR+clRRW28FtuZLwHMQMkwy/EagAAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICokRUaS03vMDExjix7oztkFS8iK/xBwKRqNhitbPNPF+wTBGOHC22hjpHqXEYfG+nhQPH7uJ5wJbP2DAl9galGSRrU88oORuYU5SwELAy1HhOjmyGtWjVXVVeka1OOU7fQUDp9A0eUIjJnFXMXxgjbK69Tkw/z18BjCFDud9+ahfewYmQ+b4AAAA=[0m[Pipeline] sh
+ sed -i $'s/\r//g' /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4AxFqWBJzJMrYG9b9wTmilC1ocDM3blJaPueGHmKGRfNAAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKCoERUaS03vMDExjix7oztkFS8iK/xBwKRqNhitbPNPF+wTBGOHC22hjpHqXEYfG+nhQPH7uJ5wJbP2DAl9galGSRrU88oORuYU5SwELAy1HhOjmyGtWjVXVVeka1OOU7fQUDp9A0eUIjJnFXMXxgjbK69Tkw/z18BjCFDud9+ahfeAtq90r4AAAA=[0m[Pipeline] sh
+ chmod a+x /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4Kdz+C3jWQIpr/ViJFTGGu4HDkiXDjnf//kiVyeBCJZNAAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKClBJR0VppeIFJjLFj3Tm2g1PxIr7GHwhEomKL1c4283zBOgY4ctBoFfWGYmvQu1HPCzOH/uo4o+ULtkyRncJG5YY7dVqw4aRgSVHCSsBGUes4GtIJtsLKu6ycJF2dU5i/g4DSdAM8oBCzOcmQvjAF2N3GLjL9PH8FMPkEZb3/VO3fQ+umy74AAAA=[0m[Pipeline] sh
+ sh /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
/root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh: line 7: [: too many arguments
[8mha:////4CJBBfZWjxe9UGIGYSKNBEab8Cvgxv9EsX4u6U8l9+zvAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKCBB2iSmu54QUmNsaJdWdsh6TiRXyNPxCIRMUWq51t5vmCZYpw5GixNdQ5So3D4Hs7LRw4dhfPA7Z8xoYpsTcozSBZm3pGydnAnKKEhYCVocZzcmQzrEWr7qryimx1ynH6DgJKp2/wgEJM5qxi/sIYYXPtdWL6ef4KYAwhQ7nbf2r7BgSW0ii+AAAA[0m[Pipeline] }
[8mha:////4BJ3niM699yfMaO4XlYckMNVxHBxF/A/6tXrT01aXGOTAAAAph+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigSgQSkVrueEFJjHGiXUXbAen4kV8jT9gEYmKLVY728zrDcvgoWZvsNPUWwqNxcGNJi9M7Pur44QdX7BhCuw0Sp0kt/o0o+SoYU5RwkLASlPjOFgyEdaiUw9VOUWmOkefv6OA0rZ3eEIhsjkqH78wedjcxjYw/Tx/BTANQ4Ryt8+1PXwARMxJg74AAAA=[0m[Pipeline] // stage
[8mha:////4COicnFv5cVB+y+WjajJHVH1ToABozYqe2wwQhbZhHHzAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIpQDSIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsXZLsPLn/NEL0AAAA=[0m[Pipeline] }
[8mha:////4MM3ujN+aYRxPvogsEC0OhvZXzOBlOHuSZFusz65OCwVAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiIUFBaOlW2GxhNEiDGQ2cVkESpP5NW8g4zMWPmr/17zXm+sU8SRo1Otpc5Tqr3qw+Dmp0aO3TXwqFq+qJopcbCqsmPFjT0tWLFYLMtyrDQ2lurAyZMTbHVrHqYIhlxxlji7g0bumzueyPRcFhPlC1PE7jY0ienX+RvA1PeCvNwLsvIDPcPJs70AAAA=[0m[Pipeline] // node
[8mha:////4Hb458bE9p/rw4BI2anzZ7fRo14kIciOt5qCjNc9pjKKAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKRoEFCVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5Lu9INt+AHpknka9AAAA[0m[Pipeline] End of Pipeline
Finished: SUCCESS
